package.json

```diff
@@ -297,6 +251,57 @@
             "markdownDescription": "Specifies whether to see only net new issues or known issues. Only applies to Code Security and Code Quality.\n\nNote: this is an experimental feature. Please reach out to [support.snyk.io](https://support.snyk.io) for more details."
           }
         }
+      },
+      {
+        "id": "onboarding",
+        "title": "Onboarding",
+        "order": 5,
+        "properties": {
+          "snyk.yesWelcomeNotification": {
+            "//": "Name starts with y to put it at the end, as configs are sorted alphabetically",
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "Show welcome notification after installation and restart",
+            "scope": "application"
+          },
+          "snyk.trustedFolders": {
+            "type": "array",
+            "default": [],
+            "description": "Folders to trust for Snyk scans."
+          }
+        }
+      },
+      {
+        "id": "cli",
+        "title": "Snyk CLI",
```

Comment:
LS is not CLI - I'd rather do it like above.

Reply:
renamed to "CLI & Language server"